data/ai/utils/custom_command.cfg

```diff
@@ -0,0 +1,18 @@
+
+#define ENABLE_AI_COMMAND_RABBIT_SPAWN
+[lua]
+code = <<
+
+wesnoth.register_synced_command("rabbit_despawn", function (cfg)
```

Comment:
These should probably be registered when the MicroAI is enabled, which would mean somewhere in _data/ai/micro_ais/mai_defs/animals.lua_.

Reply:
ye i know nothing about ai code, someone else can do that.